apps/condo/domains/common/components/UseDeskWidget.tsx

```diff
@@ -4,7 +4,11 @@ import getConfig from 'next/config'
 import get from 'lodash/get'
 
 const { publicRuntimeConfig:{ UseDeskWidgetId } } = getConfig()
-const fieldId = 20560
+const useDeskFieldsIdsMap = {
+    tin: 20560,
```

Comment:
Better to put such data to .helm, in order to be able to change values without release process.

Reply:
okay, I'll move it to helm when I cut out the jivosite id from there in the next release